public/manifest/demo.json

```diff
@@ -24,5 +24,43 @@
       "display_name": "Sri Lanka",
       "isd_code": "94"
     }
+  ],
```

Comment:
It looks like we are now duplicating a ton of data for these `json` files across each env, with just the URL being duplicated.

How about rolling them up into a `v2` and `v1` partial or something like that, and then just calling that from a `manfiest.json.erb` partial?

I think we could page cache it later if this end point gets hit a lot...

Reply:
Spiked out partials in cd68eac. It's hard to compress this information any further without being clever because `v1` and `v2` are fundamentally different (only the URLs are same). Did you have something different in mind?

The number of calls for this end point is very low btw, so we shouldn't have to worry about caching for a _long_ time even if we do partials etc.